client/src/pages/private/SiteTable.js

```diff
@@ -286,49 +313,70 @@ export default ({ sites, viewOnly }) => {
         </Grid>
 
         <CheckPermissions roles={roles} permittedRoles={['ministry_of_health']}>
-          <Grid item xs={2} />
+          <Grid item xs={8} />
           <Grid className={classes.rootItem} item xs={2}>
-            <Button
-              onClick={() => {
-                setActiveModalForm('new-site');
-              }}
-              size='medium'
-              text='Create Site'
-              startIcon={<AddCircleOutlineIcon />}
-            />
+            <Box px={2} display='flex' justifyContent='end'>
+              <Button
+                onClick={openActionMenu}
+                endIcon={isActionMenuOpen ? <ExpandLessIcon /> : <ExpandMoreIcon />}
+                aria-controls='simple-menu'
+                aria-haspopup='true'
+                text='Action'
+                variant='contained'
+                fullWidth={false}
+              />
+              <Menu
+                id='action-menu'
+                anchorEl={actionMenuAnchorEl}
+                open={Boolean(actionMenuAnchorEl)}
+                onClose={closeActionMenu}
+                getContentAnchorEl={null}
```

Comment:
Is it possible to get rid of `getContentAnchorEl` instead of passing `null`?

Reply:
It must have a default value within the library, I added it to fix an issue I had with positioning https://stackoverflow.com/a/52551100